tools/gcp.py

```diff
@@ -19,16 +21,32 @@ class Config:
     gcp_project_id: str = "wandb-client-cicd"
     project: str = "nvidia-ngc-public"
     vm_image_name: str = "nvidia-gpu-cloud-image-pytorch-20220228"
+    # vm_image_name: str = "nvidia-gpu-cloud-image-tensorflow-20220228"
```

Comment:
curious why this one is commented out? :)

Reply:
oh, these defaults are never used, essentially. this just came from testing. should i rm?

Reply:
hmmm... maybe if you keep just comment about it...?